### DISCUSSION TITLE
how can I change my bot's status to active on mobile

### DISCUSSION THREAD

**Discussion by ghost** [discordjs/discord.js#5334] [Q&A] [answered]
I've searched the documentation but I didn't find anything. I have seen bots in this status like this. 
![image](https://user-images.githubusercontent.com/73111396/108171799-b8d97900-7104-11eb-8f25-545818189222.png)

**Comment by DaStormer** [comment, marked as answer]
I believe it isn't in the Discord.JS documentation.

This should work:
```js
new Discord.Client({ ws: { properties: { $browser: "Discord iOS" } }})
```

Reference: https://discord.com/developers/docs/topics/gateway#identify-identify-connection-properties
